source/common/event/libevent_scheduler.cc

```diff
@@ -16,7 +16,18 @@ void recordTimeval(Stats::Histogram& histogram, const timeval& tv) {
 } // namespace
 
 LibeventScheduler::LibeventScheduler() {
+#ifdef WIN32
+  event_config* event_config = event_config_new();
+  RELEASE_ASSERT(event_config != nullptr,
+                 "Failed to initialize libevent event_base: event_config_new");
+  // Request wepoll backend by avoiding win32 backend.
+  int error = event_config_avoid_method(event_config, "win32");
+  RELEASE_ASSERT(error == 0, "Failed to initialize libevent event_base: event_config_avoid_method");
+  event_base* event_base = event_base_new_with_config(event_config);
+  event_config_free(event_config);
```

Comment:
wepoll not supporting edge-trigger mode continues to be unfortunate, but it seems like this mode works better than the select-based version.  No objections to switching over to it.

Reply:
Agreed lack of edge-triggering is rough. This will at least provide some better scalability

Reply:
Let me know when this change is ready to move forward, the PR is current marked as "draft".

/wait